### DISCUSSION TITLE
Sponsors

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#39685] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can we get sponsors from GitHub?

**Comment by david-engelmann** [comment, marked as answer]
[You can find all of the info on Github sponsors here](https://github.com/sponsors). You can go through the approval process if the program is live in your region. Check out this screenshot to see the places its available.

![image](https://user-images.githubusercontent.com/81196676/203135550-c4d76cf4-8843-48a2-85b1-6d4c4926d482.png)

As far as actually getting sponsored, you'll need to contribute to projects, or create your own that others really enjoy.
